main.go

```diff
@@ -125,10 +128,14 @@ This directive runs executable with arguments and put its stderr and stdout outp
 	anchorDir := cmd.Flag("anchor-dir", "Anchor directory for all transformers. PWD is used if flag is not specified.").ExistingDir()
 	linksLocalizeForAddress := cmd.Flag("links.localize.address-regex", "If specified, all HTTP(s) links that target a domain and path matching given regexp will be transformed to relative to anchor dir path (if exists)."+
 		"Absolute path links will be converted to relative links to anchor dir as well.").Regexp()
-	// TODO(bwplotka): Add cache in file?
 	linksValidateEnabled := cmd.Flag("links.validate", "If true, all links will be validated").Short('l').Bool()
 	linksValidateConfig := extflag.RegisterPathOrContent(cmd, "links.validate.config", "YAML file for skipping link check, with spec defined in github.com/bwplotka/mdox/pkg/linktransformer.ValidatorConfig", extflag.WithEnvSubstitution())
 
+	cacheDisabled := cmd.Flag("no-cache", "If true, link checking will not create local SQLite cache file").Bool()
+	// Default validity is for 5 days.
+	cacheValidity := cmd.Flag("cache.validity", "Duration till which link in cache is considered valid").Default("120h").Duration()
+	clearCache := cmd.Flag("cache.clear", "If true, entire cache database will be dropped").Bool()
```

Comment:
Yes locally, but I guess we cannot clear GitHub Actions cache yet since this is open: https://github.com/actions/cache/issues/2. So flag might be useful if we want to delete cache immediately from GH action and not wait till GitHub refreshes on its own. 🙂

Reply:
Sure!